src/less/button/button.less

```diff
@@ -8,6 +8,7 @@ a.fake-btn {
 
     background-color: transparent;
     border-radius: var(--btn-border-radius, calc(@button-height-small / 2));
+    border-width: 2px;
```

Comment:
Unfortunately, this is also affecting secondary button which it should not.
This should be overriden for tertiary only.
<img width="326" alt="Screen Shot 2022-08-17 at 8 19 12 AM" src="https://user-images.githubusercontent.com/1755269/185177839-8bd3b206-1bfe-4621-85cf-79b5cbb34a8a.png">